WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsViewModel.swift

```diff
@@ -62,7 +62,7 @@ final class WooShippingCreateLabelsViewModel: ObservableObject {
 
     /// Address to ship to (customer address), formatted for display and split into separate lines to allow additional formatting.
     private(set) lazy var destinationAddressLines: [String]? = {
-        (destinationAddress?.formattedPostalAddress)?.components(separatedBy: .newlines)
+        (destinationAddress?.formattedPostalAddress)?.components(separatedBy: ", ")
```

Comment:
Just a note that this change is because the `formattedPostalAddress` for a `ShippingLabelAddress` uses new lines, whereas the `formattedPostalAddress` for a `WooShippingAddress` uses commas.